src/code/bank2.asm

```diff
@@ -152,40 +152,40 @@ ENDC
 label_002_4287:
     ld   a, [wIsShootingArrow]                    ; $4287: $FA $4C $C1
     and  a                                        ; $428A: $A7
-    jr   z, jr_002_4291                           ; $428B: $28 $04
+    jr   z, .jr_002_4291                           ; $428B: $28 $04
 
     dec  a                                        ; $428D: $3D
     ld   [wIsShootingArrow], a                    ; $428E: $EA $4C $C1
 
-jr_002_4291:
+.jr_002_4291:
```

Comment:
Thanks for turning all these labels into local labels 👍 

A small thing: technically putting a semicolon `:` after a label turn the label into an "exported" label. In this project this doesn't matter much: we export all labels anyway, and rgbds will give [higher priority](https://rednex.github.io/rgbds/rgbasm.5.html#SYMBOLS) to the dot, meaning the label will indeed be local.

But still, in this project we usually remove all semicolons after a local label.

If you feel like doing the relevant changes in a next PR (I'm fine with merging this one as-this), that'll be awesome :)

Reply:
OK did not know that. Will make a new PR.